includes/functions.php

```diff
@@ -114,33 +114,35 @@ function tsml_bounds()
  * @param array $change_log Built from tsml_get_changed_import_meetings()
  * @return string
  */
-function tsml_build_import_change_report($data_source_name, $change_log)
+function tsml_build_import_change_report($data_source_name, $change_log, $embed_in_email = false)
 {
     global $tsml_days;
-    $hdr_change = __('Change', '12-step-meeting-list');
-    $hdr_meeting_name = __('Meeting Name', '12-step-meeting-list');
-    $hdr_day_time = __('Day / Time', '12-step-meeting-list');
-    $hdr_slug = __('Meeting Slug', '12-step-meeting-list');
-    $hdr_group = __('Group Name', '12-step-meeting-list');
-    $hdr_notes = __('Notes', '12-step-meeting-list');
-    $message = __('<h2>' . $data_source_name . __(' Feed Update Report', '12-step-meeting-list') . '</h2>');
-    $message .= "<table style=\"text-align:left\" cellspacing=\"5\"><tbody><tr><th>$hdr_change</th><th>$hdr_meeting_name</th><th>$hdr_day_time</th><th>$hdr_slug</th><th>$hdr_group</th><th>$hdr_notes</th></tr>";
+    if (!$embed_in_email) {
+        $hdr_meeting_day_time = __('Meeting', '12-step-meeting-list');
+        $hdr_slug = __('Slug', '12-step-meeting-list');
+        $hdr_notes = __('Notes', '12-step-meeting-list');
+        $message = "<table style=\"margin-bottom:10px;text-align:left;border:1px solid #dddddd;padding: 8px;\" cellspacing=\"5\">";
+        $message .= "<tr><th>$hdr_meeting_day_time</th><th>$hdr_slug</th><th>$hdr_notes</th></tr>";
+    } else {
+        $message = "<table style=\"width:100%;margin-bottom:10px;text-align:left;border:1px solid #dddddd;padding: 8px;\" cellspacing=\"5\">";
+    }
+
```

Comment:
could we make the appearance the same between the email and when showing it in debug mode? want to try to keep logic to a minimum

Reply:
debug_mode is for the troubleshooter, whereas the email let's the local admin know what's going on. Two different but similar purposes where a common routine work's for both.

Reply:
Need the slug for trouble-shooting....

Reply:
Reminder that the default setting for $tsml_debug is false, so normal situation is just the progress bar will be visible during an Update operation.

Reply:
@joshreisner, I've removed report title and column headers from the debug report to match up with the email report. Only difference remaining is that the Slug is available in the on-screen Debug Report. Can you live with that?

Reply:
i dont see why they should be different though - for example in the line above, it looksl ike the only difference is if it's embedded in email it gets 100% width, otherwise width is not specified? seems like not enough of a difference to have separate logic

for slugs, does that help you look up the meeting? could we link the meeting name instead?

the usage of the arrow is a little strange, could we take that out? it implies that there is a progression, whereas i think you're just using it as a separator

Reply:
- I didn't like the 100% width on the debug screen; now that I think about it, I don't think the email needs it either.
- Thanks for the link idea, that will be much better.
- I liked the arrow separator but sort of expected you wouldn't. Will remove.